### PR TITLE
Release 3.11.0

### PR DESCRIPTION
**Pull request by ycjcl868**
🎉🎉🎉

### CODE REVIEW

**Comment by ycjcl868** [comment]
@ilanus added.

**Comment by zombieJ** [comment]
这个 CI 看一下： https://circleci.com/gh/ant-design/ant-design/29819?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Comment by ycjcl868** [comment]
ci cache？ @chenshuai2144 . I run `rm -rf node_modules && npm test -- -w 1` in local env, it's ok.

image

**Comment by ycjcl868** [comment]
I add some test cases to increase the cov

**Comment by ycjcl868** [comment]
ci passed

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -15,6 +15,42 @@ timeline: true
 
 ---
 
+# 3.11.0
+
+`2018-12-02`
+
+3.11.0 brings two new Component, and a lot of exciting changes and new features.
```
Comment (ilanus) [inline CHANGELOG.en-US.md]:
```diff
- new Component,
+ new Components
```

Comment (ycjcl868) [reply CHANGELOG.en-US.md]: done, thanks!

File: CHANGELOG.en-US.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ timeline: true
 
 `2018-12-02`
 
-3.11.0 brings two new Component, and a lot of exciting changes and new features.
+3.11.0 brings two new Components, and a lot of exciting changes and new features.
```
Comment (ztplz) [inline CHANGELOG.en-US.md]: I prefer to `3.11.0 brings two new Components, a lot of exciting changes and new features`.

**Review by ztplz — APPROVED** [review]

File: CHANGELOG.en-US.md (hunk 2 above)
Comment (ycjcl868) [reply CHANGELOG.en-US.md]: `3.11.0 brings two new Components and a lot of exciting changes and new features.`

**Review by ztplz — APPROVED** [review]

File: CHANGELOG.en-US.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ Component Fixes / Enhancements:
 - 🌟 Upgrade `rc-notification` to `3.3.0`, Notification Added `onClick` prop that is called when the notification is clicked. [#11832](https://github.com/ant-design/ant-design/issues/11832)
 - Transfer
   - 🌟 Added `onSearch` prop that is executed when search field are changed and deprecated `onSearchChange`. [#12422](https://github.com/ant-design/ant-design/pull/12422)
-  - 🌟 Added `disabled` prop that whether disabled transfer. [#13330](https://github.com/ant-design/ant-design/issues/13330)
+  - 🌟 Added `disabled` prop that whether disable transfer. [#13330](https://github.com/ant-design/ant-design/issues/13330)
 - 🌟 Refactor Badge, support `count` as custom component. [#12140](https://github.com/ant-design/ant-design/pull/12140) [@supra28](https://github.com/supra28)
 - 🌟 Slider Added `tooltipVisible` prop that whether Tooltip will show always. [#12915](https://github.com/ant-design/ant-design/pull/12915) [@zy410419243](https://github.com/zy410419243)
```
[the remark this answers is not in the document]
Comment (ycjcl868) [reply CHANGELOG.en-US.md]: done.

File: CHANGELOG.en-US.md (hunk 2 above)
Comment (ztplz) [reply CHANGELOG.en-US.md]: But usually one `and` is enough.
For example:
``````
I have a banana, an orange and some apples.
``````

Comment (ycjcl868) [reply CHANGELOG.en-US.md]: `3.11.0 brings two new Components, a lot of exciting changes and new features.`

Comment (ycjcl868) [reply CHANGELOG.en-US.md]: done

**Pull request merged**